exercises.md

```diff
@@ -71,7 +77,7 @@ the energy radiated from the Earth as a function of its temperature (in Kelvin)
 
 The `plot_temperature_vs_emissivity()` and `plot_temperature_vs_albedo()` functions generate matplotlib plots and use a custom `linspace` function
 in `helpers.py` where `linspace(x, y, n)` generates a NumPy of `n` floating
-point values from `x` to `y` inclusive, evenly spaced, e.g., 
+point values from `x` to `y` inclusive, evenly spaced. Now for the `linspace` function:
```

Comment:
It might have been just me but initially I got a bit confused that the test exercises below should be done for `linspace` function (as opposed to `plot_temperature...`). Hence I tried to emphasise it a bit.